cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,109 @@
+/* eslint-disable max-len */
+/* eslint-disable indent */
+/* eslint-disable cypress/no-unnecessary-waiting */
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
+  beforeEach(() => {
+    cy.visit('/');
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+  });
+
+  it('should contain pagination', () => {
```

Comment:
You can make a task or a command to create a user, and then here create a loop to create three users in a loop. It will save you some lines in your code and it will be easier to support.